.github/workflows/distribute-grain.yml

```diff
@@ -0,0 +1,61 @@
+name: Distribute Grain
+on:
+  # A new Cred interval every Sunday. We'll distribute Grain 5 minutes
+  # after the new interval
+  schedule:
+    - cron: 5 0 * * 0 # 00:05 UTC, 16:05 PST
+  push:
+    branches:
+      - "graintest-*" # won't actually update ledger or deploy frontend
```

Comment:
It says we won't actually update ledger or deploy frontend. Is this accurate? Also I believe now we never update the frontend directly in this action?

Now that we are switching to a "generates PR" approach, I'm happy to have it also trigger on pushes to graintest-. We can see that the PR was generated, and then close it :)

Reply:
let's rename the branch prefix to `grain-trigger-*`